### ISSUE TITLE
[Bug] "View visitor profile" link not accessible via keyboard

### ISSUE THREAD

**Issue by diosmosis** [piwik/piwik#23140]
### What happened?

The "View visitor profile" link in the visitor log cannot be tabbed to, which means blind users cannot click it and access the visitor profile (see https://wordpress.org/support/topic/problem-with-open-visitor-profile-with-screen-reader/).

### What should happen?

It should be possible to get to the link via a screen reader and click it.

### How can this be reproduced?

1. open the visits log locally (with the visitor profile enabled)
2. try to tab through the visits log
3. notice that the rows in the visits log are skipped entirely and the View visitor profile link is not accessible

### Matomo version

5.3.0

### What browsers are you seeing the problem on?

Microsoft Edge

### Relevant log output

```shell

```

### Validations

- [x] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [x] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [x] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.
